e2e/suite_test.go

```diff
@@ -15,10 +19,18 @@ import (
 	"github.com/barkimedes/go-deepcopy"
 )
 
+type getNodeBootstrapping func(ctx context.Context, nbc *datamodel.NodeBootstrappingConfiguration) (*datamodel.NodeBootstrapping, error)
+
+var e2eMode string
```

Comment:
can we move this to another file as well? maybe flags.go?

Reply:
Moved getNodeBootstrapping, but e2eMode - I'd prefer to leave it here - it's only used in suite_test.go to get appropriate bootstrapping func and suite_test.go is also where the flag value is retrieved.

Reply:
makes sense, either way is fine with me